scylla-cdc-base/src/main/java/com/scylladb/cdc/transport/WorkerTransport.java

```diff
@@ -13,5 +13,13 @@
     Map<TaskId, TaskState> getTaskStates(Set<TaskId> tasks);
     void setState(TaskId task, TaskState newState);
     void moveStateToNextWindow(TaskId task, TaskState newState);
-    boolean shouldStop();
+
+    /**
+     * @deprecated Use {@link Worker.stop()} instead")
+     * @return
+     */
+    @Deprecated
```

Comment:
I think we shouldn't deprecate this. Kafka connector will be still using this.

Reply:
Also I don't think default implementation makes sense. There are different implementations of transport and they either use the method or not. Chance the default implementation does not fit well.